prepare.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -u
+set -e
+set -o pipefail
+
+mkdir -p ./website/scripts
+rm -rf ./website/scripts/*
```

Comment:
- The folder may already exist, hence `mkdir -p`
- If the folder already exists, there may be stuff we don't want in it, and we should make sure to delete it